tsfc/driver.py

```diff
@@ -84,7 +89,7 @@ def compile_integral(integral_data, form_data, prefix, parameters, interface):
         _.update(parameters)
         parameters = _
     if interface is None:
-        interface = firedrake_interface.KernelBuilder
+        interface = firedrake_interface_loopy.KernelBuilder
```

Comment:
i.e. pass `coffee=True/False` into compile_integral and handle the relevant cases here.